libdnf/module/modulemd/ModuleDefaultsContainer.cpp

```diff
@@ -0,0 +1,78 @@
+#include "ModuleDefaultsContainer.hpp"
+#include "../ModulePackageContainer.hpp"
+
+ModuleDefaultsContainer::ModuleDefaultsContainer()
+{
+    prioritizer = std::shared_ptr<ModulemdPrioritizer>(modulemd_prioritizer_new(), g_object_unref);
+}
+
+void ModuleDefaultsContainer::fromString(const std::string &content, int priority)
+{
+    GError *error = nullptr;
+    GPtrArray *failures;
+    GPtrArray *data = modulemd_objects_from_string_ext(content.c_str(), &failures, &error);
+
+    saveDefaults(data, priority);
+    reportFailures(failures);
+}
```

Comment:
I don't know who is owner of GPtrArray failures.
Are we responsible for freeing all data `g_ptr_array_free(failures, TRUE);`
or freeing GPtrArray wrapper `g_ptr_array_free(failures, FALSE);`. 
Or it is not our job?
This is on more places.

Reply:
unless you use g_autoptr(), you should free it yourself